pkg/operator/ceph/object/bucket/util.go

```diff
@@ -136,3 +142,12 @@ func randomString(n int) string {
 	}
 	return string(b)
 }
+
+func SetBucketProvisionerThreads(c *clusterd.Context) {
+	bucketProvisionerNumThreads, err := k8sutil.GetOperatorSetting(c.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_LIB_BUCKET_PROVISIONER_THREADS", bucketProvisionerDefaultThreads)
+	if err != nil {
+		logger.Warning("failed to fetch thread count for bucket provisioner")
+	} else {
+		os.Setenv(libBucketProvisionerThreads, bucketProvisionerNumThreads)
```

Comment:
Why do we need any code changes for this? If the env var to set is `LIB_BUCKET_PROVISIONER_THREADS`, then the admin could just set that env var directly on the operator deployment spec (in operator.yaml).

Reply:
Make sense and modified